debian/control

```diff
@@ -9,6 +9,7 @@ Uploaders:
 Build-Depends:
  attr,
  autoconf,
+ avahi-glib,
```

Comment:
Is this right? I'd expect it to be libavahi-glib-dev or something like that. It has to be the package that contains the avahi-glib.pc file, I believe.

Reply:
I just checked the contents of that package and you're right, it has `avahi-glib.pc`